### PR TITLE
Add endpoint for vuln errata linking

### PR DESCRIPTION
**Pull request by semtexzv**
Add new endpoints used by vulnerabilitites.
These endpoints:
- Don't support filtering, paging
- Provide just cross-linking between systems and advisories

### CODE REVIEW

File: listener/upload.go [hunk 1]
```diff
@@ -221,7 +221,7 @@ func updateSystemPlatform(tx *gorm.DB, inventoryID string, accountID int, host *
 	utils.Log("inventoryID", inventoryID, "packages", len(updatesReq.PackageList), "repos",
 		len(updatesReq.RepositoryList), "modules", len(updatesReq.ModulesList),
 		"addedRepos", addedRepos, "addedSysRepos", addedSysRepos, "deletedSysRepos", deletedSysRepos).
-		Debug("System created or updated successfully")
+		Debug("Systems created or updated successfully")
```
Comment (Josca) [inline listener/upload.go:224]: Why do you use plural here? Only one system at the time is created/updated, or not?

File: manager/controllers/systems_advisories_view.go [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+func systemsAdvisoriesQuery(acc int, systems, advisories []string) *gorm.DB {
+	return database.Db.
+		Table("system_advisories sa").
+		Select(systemsAdvisoriesSelect).
+		Joins("join system_platform sp on sp.rh_account_id = ? and sp.id = sa.system_id", acc).
+		Joins("join advisory_metadata am on am.id = sa.advisory_id").
+		Where("sa.rh_account_id = ?", acc).
+		Where("sp.inventory_id in (?)", systems).
+		Where("am.name in (?)", advisories).
+		Order("sp.inventory_id, am.id")
```
Comment (Josca) [inline manager/controllers/systems_advisories_view.go:36]: We should conditionally use "inventory.hosts" table to filter system_platform items as used in other handlers.

**Review by Josca — CHANGES_REQUESTED** [review]
Lgtm in general, just two comments or questions.

File: manager/controllers/systems_advisories_view.go (hunk 2 above)
Comment (semtexzv) [reply manager/controllers/systems_advisories_view.go:42]: Added

File: manager/controllers/systems_advisories_view.go [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+
+import (
+	"app/base/database"
+	"app/manager/middlewares"
+	"github.com/gin-gonic/gin"
+	"github.com/jinzhu/gorm"
+	"net/http"
+)
+
+type AdvisoryName string
```
Comment (Josca) [inline manager/controllers/systems_advisories_view.go:11]: why do you use these typedefs?

Comment (semtexzv) [reply manager/controllers/systems_advisories_view.go:11]: Well, it was getting hard to read the code.

Comment (Josca) [reply manager/controllers/systems_advisories_view.go:11]: For me it's even harder now. It looks like usage of new structs but it's only string. But if you like it ...

**Review by Josca — APPROVED** [review]

**Pull request merged**